ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/qml/GAnalyticsView.qml

```diff
@@ -0,0 +1,92 @@
+import QtQuick 2.0
+import Telemetry 0.1
+import QtGraphicalEffects 1.0
```

Comment:
🤡 No action needed at this point since this is already used elsewhere in the samples, but worth pointing out that QtGraphicalEffects isn't in Qt 6, so this will need to change at that point